src/components/form/_form.scss

```diff
@@ -167,20 +166,20 @@
     }
 
     &.erroneous {
-      border: 1px solid $color-red;
+      border: $border-width-base $border-style $color-red;
     }
   }
 
   &__checkboxes {
-    margin-top: $spacing-vertical;
+    margin-top: $spacing-4;
   }
 
   &__checkbox {
-    margin-bottom: $spacing-vertical-sm;
+    margin-bottom: $spacing-2;
     padding-left: 1.5em;
     position: relative;
 
-    @media screen and (min-width: map-get($bp, 'sm')) {
+    @media screen and (min-width: map-get($breakpoint, "breakpoint-1")) {
       padding-left: 1.35em;
```

Comment:
Can we replace this with something close with what we have in the spacing variables?